backend/segmentation/api/tracker_kymograph.py

```
@@ -236,44 +203,48 @@ def _filament_features(p: PolylineInput) -> _Filament:
+    Holds the last two *endpoint-aligned* observations, so a constant-velocity
+    prediction can be computed despite arbitrary centerline direction. Matching
+    a new frame against the predicted filament — rather than the raw previous
+    frame — is what makes the LAP robust to the per-frame geometry jitter of
+    independently re-detected MTs.
+
+    The EMA embedding template this used to carry is gone with the v5H swap;
+    the equivalent smoothing now comes from the motion model alone.
     """
 
-    __slots__ = ("last_frame", "last_feat", "prev_frame", "prev_feat", "emb_template")
+    __slots__ = ("last_frame", "last_feat", "prev_frame", "prev_feat")
```

Ruff (0.16.1)

[warning] `_TrackState.__slots__` is not sorted

Apply a natural sort to `_TrackState.__slots__`

(RUF023)